src/common/sidebar/components/Logs/Logs.tsx

```diff
@@ -16,11 +16,21 @@ function Logs({ logs }: LogsProps) {
       overflow="scroll"
     >
       <Box overflow="scroll">
-        <Heading alignItems="center" mx="20px" my="10px" size="md">
+        <Heading
+          as="h1"
+          textDecoration="underline"
+          alignItems="center"
+          mx="20px"
+          my="10px"
+          size="md"
+        >
+          Logs
+        </Heading>
+        <Text fontSize="md" fontWeight="bold" mx="10px" my="10px" isTruncated>
```

Comment:
We want to make this compact, so remove margins and make fontsize "xs". can you add a screenshot of what it looks like after these changes??